### DISCUSSION TITLE
pyc file can`t workon

### DISCUSSION THREAD

**Discussion by XSXIEYCH** [mitmproxy/mitmproxy#6152] [Q&A]
Hello guy
Thank you very much for providing such a perfect tool
I found some issue when I use mitmdump -s xx,pyc
The below info is errors detail
PS C:\Users\91096\Desktop\MyFile\django\test1\__pycache__> mitmdump -s .\interpat.cpython-310.pyc
[23:28:28.241] Loading script .\interpat.cpython-310.pyc
[### **23:28:28.243] in script .\interpat.cpython-310.pyc: source code string cannot contain null bytes**
![image](https://github.com/mitmproxy/mitmproxy/assets/70522038/ca3a3b70-fd9d-4f21-91df-7e5e96defedb)
Could you fixed the issue?

**Comment by XSXIEYCH** [comment]
I also found this issue on Python 3.8

**Reply by XSXIEYCH** [reply]
Why close the issue?

**Comment by petsneakers** [comment]
A file ending in `pyc` would be compiled bytecode, not a python script.
